### PR TITLE
Polkadot v1.1.0

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description

[description]

Fixes #(issue)

### Tasks
* [x] Update `chainbridge` to `polkadot-v1.1.0`
* [ ] Update `fudge` to `polkadot-v1.1.0`
* [ ] Decide on whether to go with `moonbeam/frontier` or `paritytech/frontier`
* [ ] Check if `eth_getLogs` is fixed in this `frontier` version

# Checklist:

- [ ] I have added Rust doc comments to structs, enums, traits and functions
- [ ] I have made corresponding changes to the documentation
- [ ] I have performed a self-review of my code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mustermeiszer** [comment]
NOTE: We will keep this PR stale for as long as Polkadot is not upgrade to `1.1.0`

**Comment by lemunozm** [comment]
Before going with it, can I have time to finish https://github.com/centrifuge/centrifuge-chain/pull/1609? The amount of breaking change would be really high, and I think migrating after #1609 makes things simpler, having fewer dependencies

**Comment by wischli** [comment]
> Before going with it, can I have time to finish #1609? The amount of breaking change would be really high, and I think migrating after #1609 makes things simpler, having fewer dependencies

Given that Polkadot has not even upgraded to v1.0.0 yet (emergency stopped last week due to XCM bug noticed in Kusama), I don't think this PR needs attention before EOY.

**Comment by lemunozm** [comment]
> - [ ] Decide on whether to go with `moonbeam/frontier` or `paritytech/frontier`

@mustermeiszer, could you expand on this topic? Is it related to the 4th point?

File: runtime/integration-tests/src/liquidity_pools/pallet/polkadot/setup.rs [hunk 1]
```diff
@@ -0,0 +1,143 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```
Comment (lemunozm) [inline runtime/integration-tests/src/liquidity_pools/pallet/polkadot/setup.rs:1]: Where did this file  (and kusama) version come from? Is this required now we use the `generic` module?

**Comment by lemunozm** [comment]
I am closing this. I want to submit the new changes to the a branch with the same name, but I wish to conserve this PR diffs as a reference.

**Pull request closed without merging**